tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -41,59 +59,297 @@ gcloud config set compute/zone $ZONE
+
+<details>
+<summary>Click to reveal commands</summary>
+
 ```sh
 gcloud config configurations activate [CONFIGURATION NAME] #The configuration for the project you want to use
 PROJECT=$(gcloud config get-value project)
 ```
 
-## Copy the files for this tutorial to a new working directory and git repository
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+gcloud config configurations activate [CONFIGURATION NAME] #The configuration for the project you want to use
+$env:PROJECT=$(gcloud config get-value project)
+```
+
+</details>
+
+# Task 2: Copy the files for this tutorial to a new working directory and git repository
+
+In this step, you'll be downloading the files to your local environment and initialize git the working directory to use in later steps.
+
+## Building a Linux Image
 
-1.  Create a new working directory:
+1. Create a new working directory:
 
-        mkdir helloworld-image-factory
-        cd helloworld-image-factory
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+mkdir helloworld-image-factory
+cd helloworld-image-factory
+```
 
-2.  Download the tutorial scripts:
+</details>
 
-         curl -L https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/cloudbuild.yaml >cloudbuild.yaml
-         curl -L https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/install-website.sh >install-website.sh 
+2. Download the tutorial scripts:
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+curl -L https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/cloudbuild.yaml >cloudbuild.yaml
+
+curl -L https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/install-website.sh >install-website.sh 
+```
+
+</details>
 
 3.  Initialize a git repository in the working directory:
 
-        git init
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+git init
+```
+
+</details>
+
+## Building a Windows Image
+1.  Create new working directories using Powershell:
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+New-Item -Name windows-image-factory -ItemType Directory
+
+Set-Location -Path ./windows-image-factory
+
+New-Item -Name scripts -ItemType Directory
+```
+
+</details>
+
+2.  Download the tutorial scripts to your local environment. To understand what's happening within the script files, scroll down to this [section](#windows-packer-scripts).
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+# Downloading the Cloud Build config file
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/cloudbuild.yaml" -OutFile "cloudbuild.yaml"
```

Comment:
Consider using an array of filenames and loop over it using `Foreach-Object`

Reply:
updated